includes/class-amp-theme-support.php

```diff
@@ -2314,6 +2314,12 @@ public static function prepare_response( $response, $args = [] ) {
 
 		$assets = AMP_Content_Sanitizer::sanitize_document( $dom, self::$sanitizer_classes, $args );
 
+		// Respond early with validation results if performing a validation request.
+		if ( AMP_Validation_Manager::$is_validate_request ) {
+			header( 'Content-Type: application/json; charset=utf-8' );
+			return wp_json_encode( AMP_Validation_Manager::get_validation_response_data(), JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES );
```

Comment:
No, probable not. It was originally pretty-printed because it appeared in the HTML comment and so it could be easily read. But now that this is no longer done, `JSON_PRETTY_PRINT` can be removed.

Reply:
Right, thanks for the clarification.